include/psp2/net/net_syscalls.h

```diff
@@ -14,17 +14,52 @@
 extern "C" {
 #endif
 
+typedef struct SceNetSyscallParameter SceNetSyscallParameter; // missing struct
+
+int sceNetSyscallAccept(int s, void *addr, void *addrlen);
```

Comment:
can you check the variable type?
for example this function would be
```suggestion
int sceNetSyscallAccept(int sockfd, SceNetSockaddr *addr, SceSize *addrlen);
```
or just `struct sockaddr`

if you didn't finish reversing yet, current header is okay.

Reply:
This is from the official log, but the specification may be lower than 1.69.
And in my opinion, SceNetSyacall seems to keep it concise by not including extra.
So Don't even have the `net.h` needed for SceNetSockaddr. (Actually now Have net.h, but it's from a previous commit and Can remove it)